metalearn/metafeatures/landmarking_metafeatures.py

```diff
@@ -26,17 +30,40 @@ def run_pipeline(X, Y, pipeline, n_folds, cv_seed):
     with warnings.catch_warnings():
         warnings.filterwarnings("ignore", category=RuntimeWarning)  # suppress sklearn warnings
         warnings.filterwarnings("ignore", category=UserWarning)  # suppress sklearn warnings
-        accuracy_scorer = make_scorer(accuracy_score)
-        kappa_scorer = make_scorer(cohen_kappa_score)
         cv = StratifiedKFold(n_splits=n_folds, shuffle=True, random_state=cv_seed)
-        scores = cross_validate(
-            pipeline, X.values, Y.values, cv=cv, n_jobs=1, scoring={
-                'accuracy': accuracy_scorer, 'kappa': kappa_scorer
-            }
-        )
-        err_rate = 1. - np.mean(scores['test_accuracy'])
-        kappa = np.mean(scores['test_kappa'])
-        return (err_rate, kappa)
+        results = new_cross_val(pipeline, X.values, Y.values, cv, 1)
+        return (results[0], results[1])
+
+def new_cross_val(pipeline, X, y, cv, n_jobs):
```

Comment:
Name this method `cross_validate` and make sure it has the same function signature as sklearn's cross_validate. It should take the same inputs and return the same outputs.

Reply:
Once we know that works, then we can modify our cross_validate function as needed to get the roc_auc.